src/sentry/notifications/notifications/rules.py

```diff
@@ -145,6 +145,7 @@ def get_context(self) -> MutableMapping[str, Any]:
         if not enhanced_privacy:
             context.update({"tags": self.event.tags, "interfaces": get_interface_list(self.event)})
 
+        # Do we still need this for perf alerts?
```

Comment:
This is used to populate the performance issue email template - if it's using the generic issue structure it should be able to use the code on line 167, but we'd need to remove it from `GROUP_CATEGORIES_CUSTOM_EMAIL`, and then we can delete the template. It's worth checking the debug views to make sure they look the same.

Reply:
Afaik we want to keep the custom email, although I'm not sure if we plan to switch to use `evidence_data` instead of this? cc @udameli

Reply:
we do want to keep custom email because we use some custom markup so likely won't be able to just display evidence_display directly

Reply:
for the codeblock below
```
f self.group.issue_category == GroupCategory.PERFORMANCE:
      context.update(
          {
              "transaction_data": [("Span Evidence", get_transaction_data(self.event), None)],
              "subtitle": get_performance_issue_alert_subtitle(self.event),
          },

I'm going to update evidence_data so that `transaction_data` and `subtitle` can be read directly from evidence_data

Reply:
I'm going to split changes here into a separate pr, I think other than this we should be good to go here.